recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
 {% set name = "TreeCorr" %}
-{% set version = "4.3.3" %}
+{% set version = "5.0.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 47c82716c6f5cf8a13289d61edf868c02f7087adfb93af10465963091cf46b38
+  sha256: 212a8a07bd64fbb0b197635c1f82f2a685a0d943c3b7a17b824078a8e49b8ae0
 
 build:
-  number: 1
-  skip: true  # [win]
+  number: 0
+  #skip: true  # [win]   (should work on Windows now)
```

Comment:
```suggestion
  skip: true  # [win]
  # It should work on Windows now, but need to do the following before it will work:
  # 1. get LSSTDESC.Coord onto conda-forge with win installation
  # 2. conditionally skip fitsio dependency for windows, since fitsio does not work on Windows yet.
```